driver-legacy/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import ProjectExtensions.configureJarManifest
+import ProjectExtensions.configureMavenPublication
+
+plugins {
+    id("project.java")
+    id("conventions.test-artifacts")
+    id("conventions.test-include-optionals")
+    id("conventions.testing-spock-exclude-slow")
+}
+
+base.archivesName.set("mongodb-driver-legacy")
+
+dependencies {
+    api(project(path = ":bson", configuration = "default"))
+    api(project(path = ":driver-core", configuration = "default"))
+    api(project(path = ":driver-sync", configuration = "default"))
+
+    testImplementation(project(path = ":bson", configuration = "testArtifacts"))
+    testImplementation(project(path = ":driver-core", configuration = "testArtifacts"))
+    testImplementation(project(path = ":driver-sync", configuration = "testArtifacts"))
+}
+
+configureMavenPublication {
+    pom {
+        name.set("The Legacy MongoDB Driver")
+        description.set("The Legacy MongoDB Driver")
+    }
+}
+
+// Disable the manifest for driver-legacy as its not a valid OSGI package
+configureJarManifest {
+    attributes["-nomanifest"] = true
+    afterEvaluate {
+        attributes.remove("Bundle-Version")
```

Comment:
It's surprising that these need to be removed explicitly even when `-nomanifest` is specified just above.  What happens if you don't do this?  Or does `-nomanifest` not mean what I think it does?

Reply:
In the publishing convention we set up some attributes outside of what the bnd plugin auto configures.

Reply:
See 68689fd for an improved approach.

Reply:
I like it.